floodviz/tests/test_utils.py

```diff
@@ -0,0 +1,210 @@
+import unittest
+import requests_mock
+from floodviz.utils import parse_rdb
+
+
+class TestRdbParsePeak(unittest.TestCase):
```

Comment:
See my comment from below. You don't need to test how the rest of the application will potentially be using the function. This is a general purpose function and should be able to handle any web service that returns an rdb format.